src/main/java/org/alliancegenome/curation_api/dao/base/BaseSQLDAO.java

```diff
@@ -480,15 +480,17 @@ public SearchResponse<E> searchByParams(Pagination pagination, Map<String, Objec
 										BooleanPredicateClausesStep<?> clause = p.bool();
 										if (useKeywordFields != null && useKeywordFields) {
 											clause.should(p.match().field(field + "_keyword").matching(searchFilters.get(filterName).get(field).get("queryString").toString()).boost(boost + 500));
+										} else { 
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineJavaCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line must not have trailing whitespace
<!-- __reviewdog__:ChA1ZjM4YjYwMmVkZmFlYjc5EgpjaGVja3N0eWxl -->